app/views/layouts/_menu.html.erb

```diff
@@ -0,0 +1,21 @@
+<header class="bg-gray-800 text-orange-50">
+    <nav class="h-16 flex justify-between items-center w-[96%] mx-auto">
+        <div>Stochastic Soccer</div>
+        <div class="">
+        <ul class="flex items-center gap-8">
+            <li class="">
+            <%= link_to "Standings", root_path, class: "hover:text-orange-300" %>
```

Comment:
You can redefine and override `link_to` so it always includes those classes; or you can define an app specific version, if you wanted to standardize links. Something like this:

```rb
def soccer_link_to(text, path, html_options = {})
  link_to(text, path, html_options.merge(class: "hover:text-orange-300")
end
```

Reply:
In the context of the menu this one makes total sense to me. 

For the button it gives me mixed feelings because it is not only a link but also a form.

Reply:
<img width="975" alt="image" src="https://github.com/eargollo/soccer/assets/154396/d61a5cf3-493b-4e59-8069-b8b8ece4cd7f">

Reply:
<img width="746" alt="image" src="https://github.com/eargollo/soccer/assets/154396/29bbbafa-20ee-4c74-981c-202c0a03fb55">

Reply:
I still feel that there is a lot of real state loss in there but this is something to think of at another time. 

I would like more info packed together and less spacing. I know this is easy with Tailwind.